src/main/java/com/notably/model/person/Path.java

```diff
@@ -0,0 +1,30 @@
+package com.notably.model.person;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Represent a path of a block.
+ */
+public class Path {
+    private List<String> path;
+    private boolean isAbsolute;
+
+    public Path(String args, Boolean isAbsolute) {
+        this.path = new ArrayList<>();
+        for (String obj : args.split("/", 0)) {
```

Comment:
What's the purpose of the `0` here? We shouldn't need it right?

Reply:
Wouldn't the `0` here instead make the code faulty? Since you're only splitting the first `/`

Reply:
> Hey @ljiazh3ng , thanks for the help with this!
> 
> I have approved the PR, so you can merge whenever you have updated the code. One question: How do you plan on using this `Path` class on the `Parser` layer? That is, how does error checking come into play?

The parser would use path when deleting a block and opening a block. I'll propose a change in the model manager API so that deleteBlock and openBlock function takes in a "path object' instead of a "block object".  For error checking, i am thinking that it will be done when finding the list of possibilities for the correction engine.